### DISCUSSION TITLE
Last Version Supported

### DISCUSSION THREAD

**Discussion by jbearden0** [Xpra-org/xpra#4351] [Q&A] [answered]
Long story short, I am working on a project and I am trying to get the Xpra client working on Windows 7. I know it isn't supported and I am not asking for support, but I would like to know what the last version of Xpra was that supported Windows 7 so I can give it a try. Bonus points for a link to an executable I can just install. The oldest version I can find is 5.0.8, which fails to run with a "procedure entry point AddDllDirectory could not be located" error. If it won't work then I'll go back to trying to get Cygwin working.

**Comment by jbearden0** [comment]
I was hoping the wayback machine would have something, but no dice. So if anybody could point me to an Xpra 3 windows install executable I would greatly appreciate it.

**Comment by totaam** [comment, marked as answer]
The version doesn't matter, what you want is the date of the build.
We use [MSYS2](https://www.msys2.org/) as build environment and ship their libraries.
[Supported Windows Versions and Hardware](https://www.msys2.org/docs/windows_support/) states that _Requires 64-bit Windows 8.1 / Windows Server 2012 R2._

As per [Is it possible to install MSYS2 on Windows 7?](https://stackoverflow.com/questions/75660136/) - it might be possible to create a build environment that still supports Windows 7, somehow. But probably not for long.
Your best bet is to find an installer generated before the ABI break: https://xpra.org/dists/windows/ or even https://xpra.org/vault/

**Reply by jbearden0** [reply]
Thanks for the reply. I did look in vault and on the wayback machine without success. I'll probably try harder to get cygwin working before I look into MSYS2.

**Reply by totaam** [reply]
> I did look in vault and on the wayback machine without success

Why would you need the wayback machine? The timestamps are accurate.
Any builds made prior to the ABI break should work.

> I'll probably try harder to get cygwin working before I look into MSYS2

I'm not sure this is going to work.
Cygwin is quite different and is likely to break a number of assumptions in xpra - ie: posix vs win32, for a start with paths.
